libevmasm/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ file(GLOB HEADERS "*.h")
 
 include_directories(BEFORE ..)
 add_library(${EXECUTABLE} ${SRC_LIST} ${HEADERS})
-eth_use(${EXECUTABLE} REQUIRED Eth::evmcore)
+eth_use(${EXECUTABLE} REQUIRED Dev::devcore Eth::ethcore)
```

Comment:
this was about evmcore dependency... 
now when I see this, we have to remove ethcore as well.  
in another PR maybe?